actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -98,6 +98,10 @@ module RequestForgeryProtection
       config_accessor :default_protect_from_forgery
       self.default_protect_from_forgery = false
 
+      # Controls whether URL-safe CSRF tokens are generated.
+      config_accessor :urlsafe_csrf_tokens, instance_writer: false
```

Comment:
The other `config_accessor`s in that file don't disable the `instance_writer`, but I didn't think it was worth being able to set this option in a controller, especially given it's not something we really want to keep long term in my opinion.